src/indicators/drought/ecmwf_seasonal.py

```diff
@@ -207,26 +197,38 @@ def __init__(self):
         super().__init__(
             year_min=2000,
             #TODO: include way to not make it crash with current year (2021)
-            year_max=2020,
+            year_max=2021,
             cds_name="seasonal-monthly-single-levels",
-            dataset="monthly_mean",
+            dataset=["monthly_mean"],
             dataset_variable_name="product_type",
         )
 
     def download(
         self,
         country_iso3: str,
         area: Area,
-        # leadtimes: List[int],
+        leadtimes: List[int] = DEFAULT_LEADTIMES,
         version: int = DEFAULT_VERSION,
         split_by_month: bool = True,
     ):
         logger.info(
             f"Downloading ECMWF seasonal forecast v{version} for years {self.year_min} - {self.year_max}"
         )
+        current_date=datetime.datetime.now()
         month_range = range(1, 13) if split_by_month else [None]
         for year in range(self.year_min, self.year_max + 1):
             logger.info(f"...{year}")
+            if split_by_month:
+                if year<current_date.year:
+                    month_range = range(1, 13)
+                elif year == current_date.year:
+                    #forecast becomes available on the 13th of the month
+                    max_month=current_date.month if current_date.day>=13 else current_date.month-1
+                    month_range = range(1,max_month+1)
+                elif year> current_date.year:
+                    logger.info(f"Cannot download data for {year}, because it is in the future")
+            else:
+                month_range = [None]
 
```

Comment:
@turnerm do you think this is a neat enough way to account for not all months being available for the current year or would you know a programmatically better way?